build.sbt

```diff
@@ -23,16 +23,11 @@ lazy val microservice = Project(appName, file("."))
     scoverageSettings,
     scalaSettings,
     scalaVersion := "2.13.12",
-    // To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
-    libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always,
     defaultSettings(),
     majorVersion := 2,
     PlayKeys.playDefaultPort := 8098,
     libraryDependencies ++= AppDependencies(),
-    scalacOptions ++= Seq(
-      "-Wconf:src=routes/.*:s",
-      "-Wconf:cat=unused-imports&src=views/.*:s"
-    )
+    scalacOptions += "-Wconf:src=routes/.*:s"
```

Comment:
Does this mean we don't compile the views any more?

Reply:
The views or html are only required for Frontend repos. I'm not sure why the views where added here in the first place.

When upgrading to play 3.0, you will find the following in the documentation...
// only required for frontends
scalacOptions += "-Wconf:cat=unused-imports&src=html/.*:s", 
// for all services
scalacOptions += "-Wconf:cat=unused-imports&src=routes/.*:s",

Reply:
we should have the unused imports bit still in there, then, shouldn't we?

Reply:
Yes, good shout.